crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1828,7 +1822,7 @@ impl SymbolicationActor {
     async fn load_cfi_caches(
         &self,
         scope: Scope,
-        requests: &[(CodeModuleId, &RawObjectInfo)],
+        requests: &[(DebugId, &RawObjectInfo)],
         sources: Arc<[SourceConfig]>,
     ) -> Vec<CfiCacheResult> {
         let futures = requests.iter().map(|(module_id, object_info)| {
```

Comment:
nit: could we do the same here as well? it _seems_ like this name is coupled with the old type (`module_id`, `CodeModuleId`) but i could be wrong here.